RP Cloud Overview and Security doc (#1504)
redpanda-data/documentation: docs/deploy/deployment-option/cloud/index.mdx

---

> title: Redpanda Cloud Docs

Reviewer: reminder that this old index needs to be replaced or updated!

Author: The header info here came up in a merge conflict and I chose the wrong option. Reverted the first 10-11 lines back. However, the rest of the updates here are intended.

    <link rel="canonical" href="https://docs.redpanda.com/docs/deploy/deployment-option/cloud/" />
</head>

> - [Redpanda Cloud Overview](../cloud/cloud-overview)

Reviewer: This index page doesn't align with the sidebars, which shows:
- Redpanda Cloud Overview
- Quickstart
- Provision a Cloud Cluster
- Security
- Networking
- Create a Topic
- Managed Connectors

>     - An overview of the Redpanda Cloud, which employs a control plane and data plane architecture that ensures no human has direct access to Redpanda brokers or customer data.

Reviewer: ```suggestion
    An overview of the Redpanda Cloud, which employs a control plane and data plane architecture that ensures no human has direct access to Redpanda brokers or customer data.
```

>     - An overview of the Redpanda Cloud, which employs a control plane and data plane architecture that ensures no human has direct access to Redpanda brokers or customer data.

Reviewer: ```suggestion
    An overview of Redpanda Cloud, which employs a control plane and data plane architecture to ensure no human has direct access to Redpanda brokers or customer data.
```

- [Provision Cloud Clusters](../cloud/cloud-overview)

>     A high-level overview of the cloud cluster provisioning options that Redpanda Cloud offers.

Reviewer: ```suggestion
    A high-level overview of the cluster provisioning options that Redpanda Cloud offers.
```

  - [VPC Peering](../cloud/vpc-peering)

    A VPC peering connection is a networking connection between two VPCs. This connection allows the VPCs to communicate with each other as if they were within the same network. A route table routes traffic between the two VPCs using private IPv4 addresses.

- [Redpanda Dedicated Cloud](../cloud/dedicated)

    Create a Redpanda Dedicated Cloud cluster in AWS and configure a VPC peering connection.

- [Redpanda BYOC](../cloud/byoc)

    Create a Redpanda BYOC cluster in your own AWS or GCP cloud.
    To create a Redpanda cluster in your own AWS cloud, log in to the Redpanda Cloud UI and create a namespace, then follow the steps for creating a Bring your own cloud (BYOC) cluster.

- [Create a BYOC Cluster on GCP](../cloud/create-byoc-cluster-gcp)

    To create a Redpanda cluster in your own GCP cloud, log in to the Redpanda Cloud UI and create a namespace, then follow the steps for creating a Bring your own cloud (BYOC) cluster.

- [Create a Topic](../cloud/create-topic)

    Create a topic for your cluster.

> - [Managed Connectors](../cloud/managed-connectors/)

Reviewer: see preview, this isn't rendering correctly. I think just remove the final /
